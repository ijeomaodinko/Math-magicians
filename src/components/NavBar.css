* {
  box-sizing: border-box;
  font-family: Verdana, Tahoma, sans-serif;
}

nav {
  background-color: rgb(8, 9, 68);
  display: flex;
  width: 100%;
  height: 100px;
  padding-bottom: 10px;
  margin-bottom: 50px;
}

h1 {
  color: white;
  font-size: 2.9rem;
  margin-top: auto;
  margin-bottom: auto;
}

.menuList {
  display: flex;
  flex-direction: row;
  font-size: 1.5rem;
  gap: 5rem;
  margin-left: 20rem;
}

ul {
  list-style: none;
}

li {
  color: white;
  text-decoration: none;
  margin-top: 12px;
  margin-bottom: auto;
}

.menui {
  color: white;
}

.menui:hover {
  color: rgb(175, 185, 35);
}

.menui:active {
  color: rgb(228, 31, 179);
}

h2 {
  font-size: 1.9rem;
}

.page {
  width: 60%;
  text-align: center;
  margin-left: auto;
  margin-right: auto;
  line-height: 60px;
}

.quote {
  width: 60%;
  text-align: center;
  margin-left: auto;
  margin-right: auto;
  line-height: 60px;
  font-size: xx-large;
}

.quoteMath {
  font-size: xx-large;
  font-weight: 900;
}

.vertical {
  border-right: 1px solid white;
  height: 10px;
  left: 50%;
}
