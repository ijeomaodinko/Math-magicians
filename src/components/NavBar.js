import React from 'react';
import { Link } from 'react-router-dom';

class NavBar extends React.PureComponent {
  render() {
    return (
      <nav className="mNav">
        <h1 className="menuTitle">
          Math Magicians
        </h1>
        <div className="menuContainer">
          <ul className="menuList">
            <li><Link className="menui vertical" to="/">Home</Link></li>
            <li><Link className="menui vertical" to="/Calculator">Calculator</Link></li>
            <li><Link className="menui vertical" to="/Quotes">Quotes</Link></li>
          </ul>
        </div>
      </nav>
    );
  }
}

export default NavBar;
