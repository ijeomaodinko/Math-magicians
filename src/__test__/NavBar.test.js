import React from 'react';
import { ReactDOM } from 'react';
import renderer from 'react-test-renderer';
import NavBar from '../components/NavBar';

it('matches snapshot', () => {
    const tree = renderer.create(<NavBar></NavBar>).toJSON();
    expect(tree).toMatchSnapshot()
});