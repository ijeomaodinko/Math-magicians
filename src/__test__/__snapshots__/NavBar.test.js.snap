// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="mNav"
      >
        <h1
          class="menuTitle"
        >
          Math Magicians
        </h1>
        <div
          class="menuContainer"
        >
          <ul
            class="menuList"
          >
            <li>
              <a
                class="menui vertical"
                href="/"
              >
                Home
              </a>
            </li>
            <li>
              <a
                class="menui vertical"
                href="/Calculator"
              >
                Calculator
              </a>
            </li>
            <li>
              <a
                class="menui vertical"
                href="/Quotes"
              >
                Quotes
              </a>
            </li>
          </ul>
        </div>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav
      class="mNav"
    >
      <h1
        class="menuTitle"
      >
        Math Magicians
      </h1>
      <div
        class="menuContainer"
      >
        <ul
          class="menuList"
        >
          <li>
            <a
              class="menui vertical"
              href="/"
            >
              Home
            </a>
          </li>
          <li>
            <a
              class="menui vertical"
              href="/Calculator"
            >
              Calculator
            </a>
          </li>
          <li>
            <a
              class="menui vertical"
              href="/Quotes"
            >
              Quotes
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
